public/js/add_event_script.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+$( document ).ready(function() {
+    $("#mixDesk").hide();
```

Comment:
@daria-sieroshtan 
JS is for dynamic stuff, and this can be done by static CSS `display: none` upon page load